carbonmark-data/components/charts/helpers/DataTable/configurations/KlimaRetirementsTableConfiguration.tsx

```diff
@@ -0,0 +1,91 @@
+import { formatTonnes } from "@klimadao/lib/utils/lightIndex";
+import { t } from "@lingui/macro";
+import PolyscanLink from "components/charts/helpers/PolygonscanLink";
+import VerraProjectLink from "components/charts/helpers/VerraProjectLink";
+import { helpers } from "lib/charts";
+import { queryRawKlimaRetirements } from "lib/charts/queries";
+import { RawRetirementsItem } from "lib/charts/types";
+import { currentLocale } from "lib/i18n";
+import layout from "theme/layout.module.scss";
+import AbstractTableConfiguration from "./AbstractTableConfiguration";
+import styles from "./styles.module.scss";
+import { Columns } from "./types";
+
+export default class KlimaRetirementsTableConfiguration extends AbstractTableConfiguration<RawRetirementsItem> {
+  fetchFunction(page: number) {
+    return queryRawKlimaRetirements({
+      sort_by: "retirement_date",
+      sort_order: "desc",
+      page_size: 10,
+      page,
+    });
+  }
+  getColumns(locale: string): Columns<RawRetirementsItem> {
+    return {
+      beneficiary: {
+        header: t`Beneficiary address`,
+        cellStyle: layout.textLeft,
+        dataKey: "beneficiary",
+        formatter: (x: string | number) => x,
+      },
+      project_id: {
+        header: t`Project`,
+        cellStyle: layout.textCenter,
+        dataKey: "project_id",
+        formatter: (projectId: string) => (
+          <VerraProjectLink projectId={projectId}></VerraProjectLink>
+        ),
+      },
+      token: {
+        header: t`Pool`,
+        cellStyle: layout.textCenter,
+        dataKey: "token",
+        formatter: (x: string | number) => x,
+      },
+      retirement_date: {
+        header: t`Date`,
+        cellStyle: layout.textRight,
+        dataKey: "retirement_date",
+        formatter: helpers.formatDateAsDaysShort(locale),
```

Comment:
This format doesn't match the figma format of yyyy-MM-dd

Reply:
Yeah I know, but I am waiting for @theprofessora advice on enforcing figma formats or using the browser localisation capabilities. See: https://github.com/KlimaDAO/klimadao/issues/1504 (In france for instance we use DD-MM-YYYY and the american format gives us headaches)